### PR TITLE
Update code sample

### PR DESCRIPTION
**Pull request by burkeholland**

### CODE REVIEW

File: docs-ref-services/logic-apps.md [hunk 1]
```diff
@@ -28,17 +28,27 @@ The advantages of using Logic Apps include the following:
...
+  .interactiveLogin()
+  .then(credentials => {
+    const client = new LogicManagement(credentials, 'subscription-id');
+    return client.workflows.listBySubscription();
+  })
+  .then(workflows => console.log(workflows));
+```
 
-##Samples
+## Samples
```
Comment (clarkio) [inline docs-ref-services/logic-apps.md]: Needs one more `#`

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/logic-apps.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-14/docs-ref-services/logic-apps.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707122200491620-76%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/logic-apps.md [hunk 2]
```diff
@@ -28,17 +28,27 @@ The advantages of using Logic Apps include the following:
 
 Logic Apps is a fully managed iPaaS (integration Platform as a Service) allowing developers not to have to worry about building hosting, scalability, availability and management. Logic Apps will scale up automatically to meet demand.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure Logic Apps modules for Node.js
-
-### Management
+#### Install npm modules
```
Comment (johnpapa) [inline docs-ref-services/logic-apps.md]: 3 #'s here

File: docs-ref-services/logic-apps.md [hunk 3]
```diff
@@ -28,17 +28,27 @@ The advantages of using Logic Apps include the following:
...
-##Example
+### Example
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const LogicManagement = require('azure-arm-logic');
+
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const client = new LogicManagement(credentials, 'subscription-id');
```
Comment (johnpapa) [inline docs-ref-services/logic-apps.md]: define a `subscriptionId` to be consistent moving forward. i think separate variables are clearer

File: docs-ref-services/logic-apps.md (hunk 1 above)
Comment (johnpapa) [inline docs-ref-services/logic-apps.md]: yes

**Comment by burkeholland** [comment]
@johnpapa All changes implemented

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/logic-apps.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-14/docs-ref-services/logic-apps.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707122221234941-76%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@burkeholland  Merged

**Pull request merged**